### ISSUE TITLE
References pane blank

### ISSUE THREAD

**Issue by rccordell** [Zettlr/Zettlr#1980]
Hello, I am excited to get started with Zettlr, particularly the integrated citations through Zotero. This software seems to answer many of my frustrations with other markdown editors on the Mac. 

I've set up the required JSON file from Zotero for citations. Intially there were a few references with characters that were throwing Zettlr off, but I've fixed those and Zettlr now recognizes my citations in text. When I type `@` it begins to search my citations, so I know it's working. However, the "References" pane in the sidebar is blank, it says "There are not citations in this document." I'm not sure what could be happening—any help would be much appreciated. In the attached screenshot you can see both a citation in text, recognized, and the empty references pane. 

As a small side question, the sidebar overlaps the text, though in screenshots it seems to be to the right of the text—should it be behaving differently?

<img width="927" alt="Screen Shot 2021-05-18 at 22 02 20" src="https://user-images.githubusercontent.com/489834/118745810-0315e780-b825-11eb-8b70-d5284571db5e.png">

**Comment by nathanlesage** [comment]
Did you save the file? In 1.8.x the references only get updates during saving (because I was afraid of lags). If that doesn't help there might be something off, in that case it would be cool if you could activate the debug mode in the preferences and then (from the newly appearing menu) select "Toggle DevTools" (or similar) and see if in the console something red appears. These devtools are the same as in Google Chrome:

![image](https://user-images.githubusercontent.com/17251683/118765894-fd97bc00-b87b-11eb-91ea-64d55c77c9be.png)

(This is a screenshot from GitHub, but you can see one error)

And concerning the sidebar, yes I thought it looked cool back when I implemented it, but in 2.x it's a normal sidebar that doesn't overlap the editor.

**Comment by rccordell** [comment]
Thank you! There was one citation throwing an error—no message came up in the regular interface, but it was in the console. Fixed that one and now the references are showing up fine.

**Comment by nathanlesage** [comment]
Ah perfect! Would you mind sharing the error message so that I can have a look that such errors appear in the regular interface and people don't have to use the console to find that out? :)

**Comment by rccordell** [comment]
No problem. The errors had cleared but I was able to reproduce it. Essentially, I had a citation with two references but I'd added a comma rather than a semicolon. That error was throwing off everything but no message or warning appeared to tell me, except in the console. Here's a similar console error I was able to generate by adding a comma between references. 

<img width="555" alt="Screen Shot 2021-05-19 at 10 43 05" src="https://user-images.githubusercontent.com/489834/118833238-4c028600-b88f-11eb-81b8-e5180217dbfc.png">
